src/main/java/com/rarchives/ripme/ripper/rippers/WordpressComicRipper.java

```diff
@@ -133,31 +202,78 @@ public Document getNextPage(Document doc) throws IOException {
                     throw new IOException("No more pages");
                 }
                 else {
+                    sleep(500);
                     return Http.url(nextPage).get();
                 }
             }
 
         @Override
         public List<String> getURLsFromPage(Document doc) {
             List<String> result = new ArrayList<String>();
-            if (explicit_domains.contains("www.totempole666.com") == true
-            || explicit_domains.contains("buttsmithy.com") == true
-            || explicit_domains.contains("themonsterunderthebed.net")
-            || explicit_domains.contains("prismblush.com")) {
+            if (getHost().contains("www.totempole666.com") == true
```

Comment:
`a == true` has the same boolean value as `a`

Please remove the redundant `== true` checks.